transformers/transformers_simple.py

```diff
@@ -0,0 +1,95 @@
+"""
+Optuna example for fine-tuning a BERT-based text classification model on the IMDb dataset
+with hyperparameter optimization using Optuna. In this example, we fine-tune a lightweight
+pre-trained BERT model on a small subset of the IMDb dataset to classify movie reviews as
+positive or negative. We optimize the validation accuracy by tuning the learning rate
+and batch size.To learn more, you can check the following
+documentation: https://huggingface.co/docs/transformers/en/hpo_train?
```

Comment:
Thank you for adding this comment. Minor thing
```suggestion
and batch size. To learn more about transformers' hyperparameter search, 
you can check the following documentation: 
https://huggingface.co/docs/transformers/en/hpo_train.
```